prereise/gather/demanddata/transportation_electrification/tests/test_smart_charging_integration.py

```diff
@@ -5,15 +5,15 @@
 
 from prereise.gather.demanddata.transportation_electrification import (
     const,
-    data_helper,
     smart_charging,
 )
 from prereise.gather.demanddata.transportation_electrification.data_helper import (
     generate_daily_weighting,
+    get_total_daily_vmt,
 )
 
 
-def test_smart_charging():
+def test_smart_charging_ldv_2days():
```

Comment:
We might be able to merge the three tests by looping through several parameters but not sure that's worth the effort.